internal/acceptance/sql_table_test.go

```diff
@@ -264,3 +266,81 @@ func TestUcAccResourceSqlTable_Liquid(t *testing.T) {
 		}`,
 	})
 }
+
+func constructManagedSqlTableTemplate(columnName string, typeName string, nullable bool, comment string) string {
+	return fmt.Sprintf(`
+		resource "databricks_schema" "this" {
+			name         = "{var.STICKY_RANDOM}"
+			catalog_name = "main"
+		}
+
+		resource "databricks_sql_table" "this" {
+			name               = "bar"
+			catalog_name       = "main"
+			schema_name        = databricks_schema.this.name
+			table_type         = "MANAGED"
+			properties         = {
+				this      = "that"
+				something = "else"
+			}
+
+			column {
+				name      = "%s"
+				type      = "%s"
+				nullable  = %t
+				comment   = "%s"
+			}
+			comment = "this table is managed by terraform"
+		}`, columnName, typeName, nullable, comment)
+}
+
+func TestUcAccResourceSqlTable_RenameColumn(t *testing.T) {
+	if os.Getenv("GOOGLE_CREDENTIALS") != "" {
+		t.Skipf("databricks_sql_table resource not available on GCP")
```

Comment:
nit: use `skipf(t)(msg)` which will cause debuggers to report the test as a failure.